cmake/wsrep.cmake

```diff
@@ -43,10 +35,9 @@ EXECUTE_PROCESS(
 #FILE(WRITE "wsrep_config" "Debug: WSREP_API_VERSION result: ${RESULT}\n")
 STRING(REGEX REPLACE "(\r?\n)+$" "" WSREP_API_VERSION "${WSREP_API_VERSION}")
 
-SET(WSREP_PACKAGE_VERSION "${WSREP_API_VERSION}.${WSREP_PATCH_VERSION}")
-SET(WSREP_VERSION
-    "${WSREP_API_VERSION}.${WSREP_PATCH_VERSION}.r${WSREP_PATCH_REVNO}"
-)
+SET(WSREP_VERSION "${WSREP_API_VERSION}.${WSREP_PATCH_VERSION}"
```

Comment:
It would be nice if revision somehow could be obtained from a program. It certainly should not be in the name of the package, but staying in greeting and error log would be nice.

But the question why we need both WSREP_VERSION and WSREP_PACKAGE_VERSION when they carry identical info still stays